tests/conftest.py

```diff
@@ -998,6 +1009,13 @@ def testlog(request, dvs):
     yield testlog
     dvs.runcmd("logger === finish test %s ===" % request.node.name)
 
+@pytest.yield_fixture(scope="class")
```

Comment:
I don't think it makes a difference for our purposes but we should probably keep this at the same scope as dvs since it depends on dvs.

Reply:
I do NOT want to call dvs_acl_manager fixture per method. I want it to be called once per class. Why do you think it should be at the same scope as that of dvs? I think scope for fixtures should be derived by need.

Reply:
I get what you're saying now! Let's go ahead and keep it as is then.